core/src/main/java/com/linecorp/armeria/client/AbstractClientFactory.java

```diff
@@ -42,12 +42,75 @@
         return newClient(URI.create(uri), clientType, options);
     }
 
+    /**
+     * Creates a new client that connects to the specified {@link URI} using the default
+     * {@link ClientFactory}.
```

Comment:
This Javadoc block is unnecessary and incorrect. Note that a method description is inherited from the method it overrides/implements.